changelog/fix_fix_208_update.md

```diff
@@ -0,0 +1 @@
+* [#208](https://github.com/rubocop-hq/rubocop-ast/issues/208): Update `MethodDispatchNode#block_literal?` to return true for `numblock`s. ([@dvandersluis][])
```

Comment:
Can you update the org name?

```suggestion
* [#208](https://github.com/rubocop/rubocop-ast/issues/208): Update `MethodDispatchNode#block_literal?` to return true for `numblock`s. ([@dvandersluis][])
```

Reply:
Yeah, the changelog tasks are not up-to-date with the main repo 🤦

Reply:
Fixed!

Reply:
Actually it came from my git setup, I'm still using the old remotes. However, the task still does need to be updated, it is missing bugfixes from the main repo. I'll try to tackle that.